src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include circle($stopwatch-size);
+  margin: 0 auto;
+  border: 5px solid $stopwatch-color;
+  cursor: pointer;
```

Comment:
do you need this?

Reply:
Why not?) 
There was mentioned "There are no strict design you should follow for this task, so you can choose on your own."